src/woo_publications/publications/models.py

```diff
@@ -29,6 +29,36 @@ class Publication(models.Model):
             "The information categories clarify the kind of information present in the publication."
         ),
     )
+    publisher = models.ForeignKey(
+        "metadata.organisation",
+        verbose_name=_("publisher"),
+        related_name="publication_publishers",
```

Comment:
```suggestion
        related_name="published_publications",
```
the related name is used for the reverse relation, so from the perspective of the organisation to the publications. You can usually come up with a good meaningful name.